src/pybind/mgr/dashboard/frontend/src/app/shared/components/card/card.component.html

```diff
@@ -1,7 +1,18 @@
-<div class="card shadow-sm flex-fill">
-  <h4 class="card-title mt-4 ms-4 mb-0">
-    {{ cardTitle }}
+<div class="card flex-fill"
+     [ngClass]="{'border-0': removeBorder, 'bg-color': cardType === 'Sync Status Card', 'shadow': iNeedShadow}">
```

Comment:
You took my suggestion literally!! xD Let's use a better variable name!
```suggestion
     [ngClass]="{'border-0': removeBorder, 'bg-color': cardType === 'Sync Status Card', 'shadow': shadow}">
```